docs/settings.md

```diff
@@ -252,6 +252,37 @@ HTTP/1.1 204 No Content
 
 ## Instance
 
+### GET /settings/capabilities
+
+List the activated capabilities for this instance.
+
+#### Request
+
+```http
+GET /settings/capabilities HTTP/1.1
+Host: alice.example.com
+Accept: application/vnd.api+json
+```
+
+#### Response
+
+```json
+{
+    "data": {
+        "type": "io.cozy.settings",
+        "id": "io.cozy.settings.capabilities",
```

Comment:
`_id` ?

Reply:
No, it is `id` and not `_id` here (as per the JSON-API spec).

Reply:
I can see `_id` [here](https://github.com/cozy/cozy-stack/pull/2200/files#diff-8727f40ae33bbd62bb4e35fbb99690d1R17) so I am wondering why it is sometimes referred to as `id` and sometimes as `_id`

Reply:
By convention, it's `_id` in the internal side (like when speaking with CouchDB), and `id` the jsonapi package transforms it to `id` on the serialization to match the JSON-API spec.